dobson/config.ini.tmpl

```diff
@@ -0,0 +1,8 @@
+[dobson]
+SlackApiToken = xoxb-00000000000-000000000000-AAAAAAAAAAAAAAAAAAAAAAAA
+DobsonSlackId = UCAAAAAAA
+KnownDevicesFile = dobson/devices.json
+MacAddressLogFile = dobson/mac_addresses.txt
+AllowedChannelIds = [
+    "GCAAAAAAA"
+    ]
```

Comment:
Does this need to be indented for this to work?

Reply:
Yes from testing it seems like it does. It's a bit ugly but I think it's nice to be able to json decode the list instead of manually parsing it. Open to changing it up if you think something else would work, though.